synapse/storage/events_worker.py

```diff
@@ -52,13 +52,14 @@
 
 class EventsWorkerStore(SQLBaseStore):
     def get_received_ts(self, event_id):
-        """Get received_ts (when it was persisted) for the event
+        """Get received_ts (when it was persisted) for the event. Raises an
```

Comment:
could have done with being a separate paragraph, ftr, since the first line is slightly special (https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)